CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
+## 1.0
+
```

Comment:
Maybe add a line here with `* Graduate package to 1.0.`

Reply:
Done